src/_pytest/assertion/rewrite.py

```diff
@@ -113,27 +113,14 @@ def exec_module(self, module):
         # tricky race conditions, we maintain the following invariant: The
         # cached pyc is always a complete, valid pyc. Operations on it must be
         # atomic. POSIX's atomic rename comes in handy.
-        write = not sys.dont_write_bytecode
+        write = not sys.dont_write_bytecode  # love me a double negative
```

Comment:
The comment is not necessary, is it?

Reply:
I was feeling snarky at the time and added it. Want me to remove it?

Reply:
I can see that.. :)
I'd rather not have it, since it adds no value - and gets read unnecessarily.

Reply:
Removed

Reply:
Ok, should not land in the Git history though also - i.e. please squash after approval.